Setup.sh

```diff
@@ -111,21 +118,23 @@ markStartOfBlock "Unpack dependencies"
     cp -R $BINARIES_DIR/Mac/include/ $WORKER_SDK_DIR
 markEndOfBlock "Unpack dependencies"
 
-markStartOfBlock "Copy standard library schema"
-    echo "Copying standard library schemas to $SCHEMA_STD_COPY_DIR"
-    cp -R $BINARIES_DIR/Programs/schema/* $SCHEMA_STD_COPY_DIR
-markEndOfBlock "Copy standard library schema"
+if [ ! -z "$SPATIAL_DIR" ]; then
+    markStartOfBlock "Copy standard library schema"
+        echo "Copying standard library schemas to $SCHEMA_STD_COPY_DIR"
+        cp -R $BINARIES_DIR/Programs/schema/* $SCHEMA_STD_COPY_DIR
+    markEndOfBlock "Copy standard library schema"
 
-markStartOfBlock "Copy GDK schema"
-    rm -rf $SCHEMA_COPY_DIR   2>/dev/null
-    mkdir -p $SCHEMA_COPY_DIR >/dev/null 2>/dev/null
+    markStartOfBlock "Copy GDK schema"
+        rm -rf $SCHEMA_COPY_DIR   2>/dev/null
+        mkdir -p $SCHEMA_COPY_DIR >/dev/null 2>/dev/null
 
-    echo "Copying schemas to $SCHEMA_COPY_DIR."
-    cp -R $(dirname %0)/SpatialGDK/Extras/schema/* $SCHEMA_COPY_DIR
-markEndOfBlock "Copy GDK schema"
+        echo "Copying schemas to $SCHEMA_COPY_DIR."
+        cp -R $(dirname %0)/SpatialGDK/Extras/schema/* $SCHEMA_COPY_DIR
+    markEndOfBlock "Copy GDK schema"
+fi
 
 markStartOfBlock "Build C# utilities"
-    msbuild /nologo /verbosity:minimal ./SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Mac/Improbable.Unreal.Scripts.sln /property:Configuration=Release /restore
+    msbuild /nologo /verbosity:minimal ./SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Improbable.Unreal.Scripts.sln /property:Configuration=Release /restore
```

Comment:
🎉  can also remove this whole folder now `SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Mac`